packages/tldraw/src/lib/ui/components/primitives/TldrawUiButtonPicker.tsx

```diff
@@ -132,9 +132,7 @@ export const TldrawUiButtonPicker = memo(function TldrawUiButtonPicker<T extends
 						data-testid={`style.${uiType}.${item.value}`}
 						aria-label={label}
 						value={item.value}
-						data-state={
```

Comment:
actually, now that this is a ToggleItem, we might have to, optionally also do `data-state=on|off`: https://www.radix-ui.com/primitives/docs/components/toolbar#toggleitem
i think it works without it though

Reply:
Looks like that system is still being worked on (i.e. a value for the toggle group that would be picked up by the toggle item), but if it ends up being the same logic as isactive then that would be better.

Reply:
yeah i wouldn't worry about it, this is good to land